WordPress/Classes/Uploads/UploadsManager.swift

```diff
@@ -2,9 +2,12 @@ import Foundation
 
 /// Takes care of coordinating all uploaders used by the app.
 ///
-@objc
 class UploadsManager: NSObject {
     private let uploaders: [Uploader]
+
+    /// The reason why this property is lazy is that it's basically a closure with a self reference.
```

Comment:
Are you avoiding using Reachability cause we are planning on abandoning it?

Reply:
Apologies!  I forgot to push my last commit, which made some changes to how `UploadsManager` uses reachability.